integrations/node/workload-executor.js

```diff
@@ -37,6 +37,7 @@ async function main(uri, spec) {
       try {
         await runOperations(client, results);
       } catch (err) {
+        console.log({ err });
```

Comment:
I think we actually want `console.dir(err)` here instead, unless you think seeing the wrapping `{ err: ... }` is useful for debugging?